xline/src/server/xline_server.rs

```diff
@@ -305,6 +314,7 @@ impl XlineServer {
         MaintenanceServer<S>,
         ClusterServer,
         Arc<CurpServer<S>>,
+        Arc<CurpClient>,
```

Comment:
Could we call `publish` in `init_servers`?

Reply:
No, the RPC server has not been started at this stage